docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.navigation.scss

```diff
@@ -0,0 +1,101 @@
+// ICON CONTAINERS
```

Comment:
should we actually name this file `_mixins.icons.scss`?

Reply:
I went with `_mixins.menu-icons.scss` since these icons are specifically for the navigation components.

Reply:
great!!